### PR TITLE
Add popup window

### PR DESCRIPTION
**Pull request by denigit5**
- [ ] Added the popup window to my portfolio. Please review and let me know if to make changes.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -327,6 +327,126 @@ button:hover {
...
+  padding-top: 0.5rem;
+  width: 88%;
+  left: 4.5%;
+  top: 7.6rem;
+  border-radius: 16px;
+  margin-top: -7rem;
+  align-items: center;
+  background-color: white;
+}
+
```
Comment (V-Blaze) [inline style.css:346]: - You have done great so far :+1: : generating the project section dynamically :+1: :+1: But there are som changes you still need to make before moving to the next milestone. :+1: 
- [ ] Looking at the screenshot below, you can see that the desktop pop-up doesn't look like the Figma design when on the screen size of `1450 px` and above, the language list and the button tends to flow onto the Image. I'll suggest you fix this. :+1: :+1: 

- - Screenshot:
Selection_005

- [ ] Also looking at the screenshot below, you can see that when on screen sizes below `1340 px` the image on the pop-up doesn't resize and therefore the full image is not visible, part of the image is cropped out. I'll suggest you fix this also. :+1: :+1: 

- - Screenshot:
 
Selection_006

File: script.js [hunk 2]
```diff
@@ -19,4 +19,92 @@ document.addEventListener('DOMContentLoaded', () => {
...
+    worksection.innerHTML = container01 + container02 + container03 + container04;
+  }
+  document.querySelector('.project-1').addEventListener('click', () => {
+    document.querySelector('.popup').classList.remove('popup-inactive');
+    document.querySelector('.popup').classList.add('popup');
+  });
+  document.querySelector('.project-2').addEventListener('click', () => {
```
Comment (V-Blaze) [inline script.js:94]: > When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- [ ] Looking at the video below, you can see that you have the same details and image on the pop-up for every project. that's not meant to be so. According to the requirement, the pop-up is generated dynamically depending on the project that was clicked. I'll suggest you also implement this. :+1: :smile: 

- - VIDEO REFERENCE:

https://github.com/denigit5/Portfolio/assets/107632780/b583be5c-0e1c-4c3a-b988-a399465747a4


- [ ] Also currently, clicking on the X close icon tends to refresh the page. this is not correct, the popup should close without the page being refreshed. I'll suggest you fix this. :wave: :smile:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @denigit5 
## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Correct Github flow✔
-  Readme looks profesional👋 ✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by denigit5** [comment]
- [ ] Added the suggest changes. Please review and let me know if to make changes.
- [ ] I have used different images in the popupwindow for mobile version in respective of the project being built, but I Couldn't for desktop version since the images look unpresentable while sketched.

File: style.css [hunk 3]
```diff
@@ -327,6 +332,125 @@ button:hover {
...
+  height: 813px;
+  padding-top: 0.5rem;
+  width: 88%;
+  left: 4.5%;
+  top: 7.6rem;
+  border-radius: 16px;
+  margin-top: -7rem;
+  align-items: center;
+  background-color: white;
+}
```
Comment (aamirkhan2478) [inline style.css:350]: - [ ] As mentioned by the previous reviewer :point_up: if you can see that the desktop pop-up doesn't look like the Figma design when on the screen size of `1450` px and above, the language list and the button tends to flow onto the Image. Check below :point_down:  the comparison between your design and Figma design :slightly_smiling_face: 

**Your Design**
Screenshot from 2023-08-02 16-36-03

**Figma Design**
Screenshot from 2023-08-02 17-06-29

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @denigit5 & @Chaw-Ma-Ma-Win, 



Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
✔️ Good Effort
✔️ Your design is good.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**